### PR TITLE
doc: update org management policy

### PR DESCRIPTION
**Pull request by mhdawson**
- update to reflect move to OpenJS Founation
- upate to reflect that there is no longer a CommComm director

### CODE REVIEW

**Comment by targos** [comment]
There's a typo in the name of the file.

File: GITHUB_ORG_MANGEMENT_POLICY.md [hunk 1]
```diff
@@ -1,10 +1,5 @@
 # Node.js GitHub Organization Management Policy
 
-The Node.js Foundation GitHub Organization (https://github.com/nodejs) is
```
Comment (keywordnew) [inline GITHUB_ORG_MANGEMENT_POLICY.md]: Would there be a counterpart in the new version of this doc that described the GitHub Org as a development resource? Unless this has been superseded in another doc.

**Comment by mhdawson** [comment]
@targos fixed name.

File: GITHUB_ORG_MANGEMENT_POLICY.md (hunk 1 above)
Comment (mhdawson) [reply GITHUB_ORG_MANGEMENT_POLICY.md]: @keywordnew added back some of the text.  Was not clear to me that the Foundation provides the resource so wanted to remove that, but the reference to the org is good to retain.

**Pull request merged**